app/src/main/java/org/gem/indo/dooit/views/main/fragments/challenge/fragments/ChallengeRegisterFragment.java

```diff
@@ -124,6 +126,23 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         if (hasActive) {
             register.setText(getText(R.string.label_continue));
         }
+
+        if (instruction.getText() == null || instruction.getText().equals("")) {
```

Comment:
You can replace multiple null checks and equals check with 
`TextUtils.isEmpty(instruction.getText())`